lib/entities/asset.js

```diff
@@ -117,14 +117,14 @@ const ASSET_PROCESSING_CHECK_RETRIES = 5
  * @property {Asset[]} items - an array of assets
  * @property {function(): Object} toPlainObject - Returns this Asset collection as a plain JS object
  * @example
- * 
+ *
  * // require contentful-management
  * var contentfulManagement = require('contentful-management')
  * var client = contentfulManagement.createClient({
  * // This is the access token for this space. Normally you get both ID and the token in the Contentful web app
  * accessToken: 'YOUR_ACCESS_TOKEN'
- * }) 
- * 
+ * })
+ *
  * let space = await client.getSpace('SPACE_ID')
```

Comment:
Yes @TimBeyer you are right, this was me introducing this in the CDA documentations. We'll work on making all the examples using promises.